### ISSUE TITLE
Unused imports 

### ISSUE THREAD

**Issue by busunkim96** [googleapis/gapic-generator-python#169]
Flake raises warnings about unused imports.

See https://github.com/googleapis/google-cloud-python-private/pull/504
```
google/cloud/workflows_v1alpha1/services/workflows/pagers.py:2:1: F401 'copy' imported but unused
google/cloud/workflows_v1alpha1/services/workflows/client.py:3:1: F401 'typing.Mapping' imported but unused
google/cloud/workflows_v1alpha1/services/workflows/client.py:3:1: F401 'typing.Optional' imported but unused
google/cloud/workflows_v1alpha1/services/workflows/transports/grpc.py:2:1: F401 'typing.Sequence' imported but unused
google/cloud/workflows_v1alpha1/services/workflows/transports/grpc.py:2:1: F401 'typing.Tuple' imported but unused
tests/unit/workflows_v1alpha1/test_workflows.py:10:1: F401 'google.api_core.operation' imported but unused
tests/unit/workflows_v1alpha1/test_workflows.py:18:1: F401 'google.protobuf.empty_pb2 as empty' imported but unused
```

**Comment by lukesneeringer** [comment]
I added a PR (#177) for the easy ones. The two in the unit tests are actually somewhat difficult, but still worth trying to bop out.

**Comment by software-dov** [comment]
For the two in `test_workflows.py`, I don't see the corresponding imports in the `test_$service.py.j2`. Am I misreading the template? Has the issue been fixed and there's now just version shear?

**Comment by lukesneeringer** [comment]
It is rendered by the last loop, I believe.

**Comment by software-dov** [comment]
Last bit fixed by #254
